src/sentry/models/file.py

```diff
@@ -242,8 +244,8 @@ def generate_unique_path(cls, timestamp):
         return u'/'.join(pieces)
 
     def delete(self, *args, **kwargs):
-        lock = locks.get(u'fileblob:upload:{}'.format(self.checksum), duration=60 * 10)
-        with TimedRetryPolicy(60)(lock.acquire):
+        lock = locks.get(u'fileblob:upload:{}'.format(self.checksum), duration=UPLOAD_RETRY_TIME)
```

Comment:
We use the same `fileblob:upload` keyspace in deleting as in uploading. Should we consider changing this?

Reply:
No, otherwise there's a race condition.